lib/refile/backend/file_system.rb

```diff
@@ -34,8 +34,12 @@ def upload(uploadable)
         Refile.verify_uploadable(uploadable, @max_size)
 
         id = @hasher.hash(uploadable)
-        IO.copy_stream(uploadable, path(id))
-
+        begin
+          IO.copy_stream(uploadable, path(id))
+        rescue IOError
+          uploadable.open
+          retry
```

Comment:
This seems super, super evil. Hello infinite IO loop. Why would this be necessary? Also, `uploadable.open` is not part of our API contract with regards to uploadables, no idea why it's even callable here.